Simple-step-by-step-setup-instructions-for-non-Linux-users.md

```diff
@@ -0,0 +1,225 @@
+# **Simple step-by-step Supernode setup instructions for non-Linux users**
+---
+This is the formal version of the `Alpha3 Supernode Guide for Dummies` by yidakee (@el_duderino_007) dedicated to the Community Tree development branch of Graft Network.
+
+**_This is not an official guide to be used with the main development branch_**
+
+## General considerations
+
+* This is a bare minimum install recommended for linux newbies and troubleshooting purposes.
+
+* This setup is **not recommended for mainnet**, as it is very a insecure server setup.
```

Comment:
"very a" -> "a very"

Though it's not actually that insecure; it doesn't leave anything listening that doesn't need to be.